relation/models.py

```diff
@@ -0,0 +1,43 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+
+class DeliveryCenter(models.Model):
+    location = models.CharField(max_length=255)
+
+    def __str__(self):
+        return self.location
+
+
+class Technology(models.Model):
+    title = models.CharField(max_length=255)
+
+    def __str__(self):
+        return self.title
+
+
+class Member(AbstractUser):
+    delivery_center = models.ForeignKey(DeliveryCenter, on_delete=models.CASCADE, related_name='members')
+    password = models.CharField(max_length=255)
+
+
+class Position(models.Model):
+    position = models.CharField(max_length=255)
```

Comment:
The position is something static; We can have simply choice field instead of db table. It will reduce complexity.